GiftedMessenger.js

```diff
@@ -647,6 +669,7 @@ GiftedMessenger.defaultProps = {
 
 GiftedMessenger.propTypes = {
   autoFocus: React.PropTypes.bool,
+  backgroundImage: React.PropTypes.number,
```

Comment:
Should/can this be either a number (ID of the StyleSheet) or an object referring to the style itself?